lib/tools/adb-commands.js

```diff
@@ -1462,13 +1466,14 @@ methods.getDeviceProperty = async function getDeviceProperty (property) {
  * @param {string} property - The name of the property. This name should
  *                            be known to _adb shell setprop_ tool.
  * @param {string} val - The new property value.
+ * @param {boolean} privileged - Do we need to set root access for call to 'setProp'?
  *
  * @throws {error} If _setprop_ utility fails to change property value.
  */
-methods.setDeviceProperty = async function setDeviceProperty (prop, val) {
+methods.setDeviceProperty = async function setDeviceProperty (prop, val, privileged = true) {
```

Comment:
it makes sense to add a generic `opts` argument and put `privileged` there. It might be we'd need to add more parameters there